### PR TITLE
Intregration test for 2625

### PR DESCRIPTION
**Pull request by tivervac**
Tried to boil it down as much as possible. On my machine running `mvn clean verify -Dtycho-version=3.0.5` in `projects-its/issue2625/com.sigasi.hdt.test`

See #2625

### CODE REVIEW

**Comment by laeubi** [comment]
Can you please reference this in a TestClass so the test is actually executed?

**Comment by tivervac** [comment]
@laeubi Done! I've also proofread the Contributing document, mostly because I didn't know where to look for the Java test class

**Comment by laeubi** [comment]
> I've also proofread the Contributing document, mostly because I didn't know where to look for the Java test class

Many thanks, would you mind to open a separate PR for the contribution document, I think this should be merged right away and is independent from the rest.

**Comment by tivervac** [comment]
Sure, see https://github.com/eclipse-tycho/tycho/pull/2639

**Comment by tivervac** [comment]
It was available, not sure why it was failing. I've moved things around, let's see if that works better

**Comment by laeubi** [comment]
@tivervac I'm working on a fix for the problem, but the the build fails:

> The Maven artifactId (currently: "feature") must be the same as the feature ID (currently: "tycho-its-project.maven.brokenp2data.feature")

so I think a little adjustment is required here!

**Comment by tivervac** [comment]
How about this?

File: tycho-its/src/test/java/org/eclipse/tycho/test/brokenp2data/BrokenP2DataTest.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import org.junit.Test;
+
+// See #2625
+public class BrokenP2DataTest extends AbstractTychoIntegrationTest {
+
+	@Test
+	public void test() throws Exception {
+		Verifier verifier = getVerifier("brokenp2data");
+
+		verifier.executeGoals(List.of("clean", "verify", "-Dtycho-version=3.0.5"));
```
Comment (laeubi) [inline tycho-its/src/test/java/org/eclipse/tycho/test/brokenp2data/BrokenP2DataTest.java:16]:
```suggestion
		verifier.executeGoals(List.of("clean", "verify"));
```

**Pull request merged**